pom.xml

```diff
@@ -15,7 +15,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <failOnMissingWebXml>false</failOnMissingWebXml>
   </properties>
-
+  
```

Comment:
This is where I changed something (added dependencies) and then removed them.

Reply:
Please remove the extra spaces so that the file does not show up as changed at all. Also, spaces at the end of lines (even empty lines) are generally considered wrong.